code/modules/mob/dead/observer/orbit.dm

```diff
@@ -138,6 +147,24 @@
 					predators += list(serialized)
 				else if(human.faction in FACTION_LIST_ERT)
```

Comment:
Since this is a giant else if chain (which is fine - it ensures a person only appears ever in one location) the ordering of this chain needs to be altered. For example a synthetic death squad is merely listed as a synthetic: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/650bf88f-0740-43e5-9735-bd962bd7793c)

Testing may be required over multiple rounds to identify other issues with how people are listed.